README.md

```diff
@@ -1,4 +1,8 @@
-[![OpenWeave][ow-logo]][ow-repo]
+[![OpenWeave][ow-logo]][ow-core-repo]
+[![Build Status][ow-core-travis-svg]][ow-core-travis]
```

Comment:
Just an aesthetic nit: maybe add a line with only "<br><br>" on it between ow-logo and ow-core-travis-svg so there's some spacing between them in the rendered version of the README.

Reply:
I followed the identical formatting to the README in openthread. I think if we change it, we should be consistent across the projects.

Reply:
Yeah on OT the "released by Nest" tagline adds some white space to the bottom of that logo by default, so they don't appear so close together.  A better solution is probably to add a little whitespace to the top/bottom of the OW logo.

Reply:
Fixed.